bouncerscript/script.py

```diff
@@ -43,6 +43,9 @@
 
         log.info("Adding corresponding paths ...")
         for platform, path in pr_config["paths_per_bouncer_platform"].items():
+            # bug 1443104: temp hack that goes away when bug is solved
+            if "updates" in path:
+                path = path.replace("updates", "update")
```

Comment:
I'd like this to be logged to  the bouncer run log itself, (log.info(...)) to be extra clear what's happening. but otherwise sounds good as a hack to unblock beta 1

Reply:
That's done later on before making the actual call here https://github.com/mozilla-releng/bouncerscript/blob/master/bouncerscript/utils.py#L29